dashboard/app/models/user_level_evaluation.rb

```diff
@@ -33,4 +33,8 @@ class UserLevelEvaluation < StudentWorkEvaluation
   validates :student_id, presence: true
   validates :level_id, presence: true
   validates :unit_id, presence: true
+
+  def user_level_skill_evaluations
+    UserLevelSkillEvaluation.find(StudentWorkEvaluationSummary.where(student_work_evaluation_summary: self).pluck(:student_work_evaluation_id))
+  end
```

Comment:
This works but feels fishy because it seems like there should be a way to do this more aligned with rails conventions with model associations, but I'm quite sure how to say something like: 

"a UserLevelEvaluation has many UserLevelSkillEvaluations through StudentWorkEvaluationSummaries _when_ the student_work_evaluation_summary id is the id of the user_level_evaluation"  🤔